ci/common/build.bash

```diff
@@ -188,9 +188,6 @@ case "${COVERAGE_PLAN}" in
     append CMAKE_FLAGS "-DTHRUST_MULTICONFIG_ENABLE_SYSTEM_CUDA=ON"
     append CMAKE_FLAGS "-DTHRUST_MULTICONFIG_WORKLOAD=SMALL"
     append CMAKE_FLAGS "-DTHRUST_INCLUDE_CUB_CMAKE=ON"
-    append CMAKE_FLAGS "-DCUB_ENABLE_THOROUGH_TESTING=OFF"
```

Comment:
Should these flags be deleted from `ci/common/build.bash` as well?

Reply:
I'm not sure what you mean -- this patch does remove the CUB testing flags from that file.